### PR TITLE
Add Feedback mailtos to app and dashboard

### LINKED ISSUE #100: Add mailto link for providing feedback
Add to app and dashboard

Ideally pre-filled with subject line/template to help people quickly fire off an email with feedback about the app or dashboard

### PR DESCRIPTION
**Pull request by tallcoleman**
Closed #100 

Mailto link added for dashboard and app to help users report feedback.

Email is like this:

To: bikespaceto@gmail.com

Subject: BikeSpace App Feedback

> \----------
> Please describe your feedback about app.bikespace.ca below. We welcome both positive feedback (e.g. I found x feature useful) and constructive feedback (e.g. y is broken, I wish the app did z).
> 
> Especially for constructive feedback, you can help us by letting us know:
> - Your browser and platform (e.g. Safari, iPhone)
> - If it's a bug, what steps led to the problem
> - If it's something you wish the app was able to do, what goal would that feature help you accomplish? (e.g. "I think it would help advocates if you could report issues of x type", or "I wanted to be able to do step y easier on my phone")
> 
> Thank you for taking the time to help us make the app better!
> \----------
> 
> Hi BikeSpace team,

### CODE REVIEW

File: bikespace_dashboard/index.html [hunk 1]
```diff
@@ -53,6 +53,15 @@
               />Contribute</a
             >
           </li>
+          <li>
+            <a
+              href="mailto:bikespaceto@gmail.com?subject=BikeSpace%20Dashboard%20Feedback&body=----------%0APlease%20describe%20your%20feedback%20about%20dashboard.bikespace.ca%20below.%20We%20welcome%20both%20positive%20feedback%20(e.g.%20I%20found%20x%20feature%20useful)%20and%20constructive%20feedback%20(e.g.%20y%20is%20broken%2C%20I%20wish%20the%20dashboard%20did%20z).%0A%0AEspecially%20for%20constructive%20feedback%2C%20you%20can%20help%20us%20by%20letting%20us%20know%3A%0A-%20Your%20browser%20and%20platform%20(e.g.%20Safari%2C%20iPhone)%0A-%20If%20it's%20a%20bug%2C%20what%20steps%20led%20to%20the%20problem%0A-%20If%20it's%20something%20you%20wish%20the%20dashboard%20was%20able%20to%20do%2C%20what%20goal%20would%20that%20feature%20help%20you%20accomplish%3F%20(e.g.%20%22I%20wanted%20to%20see%20only%20issues%20along%20street%20x%22%2C%20or%20%22I%20wanted%20to%20better%20understand%20issues%20of%20y%20type%22)%0A%0AThank%20you%20for%20taking%20the%20time%20to%20help%20us%20make%20the%20dashboard%20better!%0A----------%0A%0AHi%20BikeSpace%20team%2C%0A%0A"
```
Comment (hulloanson) [inline bikespace_dashboard/index.html]: let's use encodeURIComponent here instead? It is kind of hard to read here

File: bikespace_frontend/src/components/FeedbackMailTo.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import {StaticImage} from 'gatsby-plugin-image';
+
+const FeedbackMailTo = () => {
+  return (
+    <a href="mailto:bikespaceto@gmail.com?subject=BikeSpace%20App%20Feedback&body=----------%0APlease%20describe%20your%20feedback%20about%20app.bikespace.ca%20below.%20We%20welcome%20both%20positive%20feedback%20(e.g.%20I%20found%20x%20feature%20useful)%20and%20constructive%20feedback%20(e.g.%20y%20is%20broken%2C%20I%20wish%20the%20app%20did%20z).%0A%0AEspecially%20for%20constructive%20feedback%2C%20you%20can%20help%20us%20by%20letting%20us%20know%3A%0A-%20Your%20browser%20and%20platform%20(e.g.%20Safari%2C%20iPhone)%0A-%20If%20it's%20a%20bug%2C%20what%20steps%20led%20to%20the%20problem%0A-%20If%20it's%20something%20you%20wish%20the%20app%20was%20able%20to%20do%2C%20what%20goal%20would%20that%20feature%20help%20you%20accomplish%3F%20(e.g.%20%22I%20think%20it%20would%20help%20advocates%20if%20you%20could%20report%20issues%20of%20x%20type%22%2C%20or%20%22I%20wanted%20to%20be%20able%20to%20do%20step%20y%20easier%20on%20my%20phone%22)%0A%0AThank%20you%20for%20taking%20the%20time%20to%20help%20us%20make%20the%20app%20better!%0A----------%0A%0AHi%20BikeSpace%20team%2C%0A%0A">
```
Comment (hulloanson) [inline bikespace_frontend/src/components/FeedbackMailTo.tsx]: encodeURIComponent?

File: bikespace_dashboard/index.html (hunk 1 above)
Comment (tallcoleman) [reply bikespace_dashboard/index.html]: That's a good idea... let me look that up

Comment (tallcoleman) [reply bikespace_dashboard/index.html]: @hulloanson  done

File: bikespace_frontend/src/components/FeedbackMailTo.tsx (hunk 2 above)
Comment (tallcoleman) [reply bikespace_frontend/src/components/FeedbackMailTo.tsx]: @hulloanson done

File: bikespace_frontend/src/components/FeedbackMailTo.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {StaticImage} from 'gatsby-plugin-image';
+
+const feedback_to = 'bikespaceto@gmail.com';
+const feedback_subject = 'BikeSpace App Feedback';
+const feedback_body = `----------
```
Comment (sonalranjit) [inline bikespace_frontend/src/components/FeedbackMailTo.tsx]: Could we follow the google typescript guide for camelcase for variable declarations. The linter should've picked these up 🤔 
https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type

File: bikespace_frontend/src/components/FeedbackMailTo.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- If it's a bug, what steps led to the problem
+- If it's something you wish the app was able to do, what goal would that feature help you accomplish? (e.g. "I think it would help advocates if you could report issues of x type", or "I wanted to be able to do step y easier on my phone")
+
+Thank you for taking the time to help us make the app better!
+----------
+
+Hi BikeSpace team,
+
+`;
+const feedback_href = [
```
Comment (sonalranjit) [inline bikespace_frontend/src/components/FeedbackMailTo.tsx]: Same thing as above about lower camelcase style

**Review by sonalranjit — CHANGES_REQUESTED** [review]
Just minor formatting for variable names

**Review by sonalranjit — APPROVED** [review]
woo thanks!

**Pull request merged**